### ISSUE TITLE
suddenly unable to mount android file system from desktop computer

### ISSUE THREAD

**Issue by dln949** [andyholmes/gnome-shell-extension-gsconnect#278]
Desktop computer:  Ubuntu (Bionic) 18.04.1, GNOME Shell 3.28.3, GSCONNECT v13.   

Android devices:  kde connect 1.8.4.

I have four Android devices, all on the same local wifi, up until today gsconnect worked fine for all four of them.  

Today, for reasons I cannot figure out, for one of the android devices, at the desktop computer I am unable to mount the android device's filesystem using gsconnect.  

The problem android device is a Samsung Galaxy Tab S2 model SM-T813 running Android 8.1.0/LineagOS 15.1-20181022-NIGHTLY-gts210vewifi.  

Everything else seems to be working fine in gsconnect for this particular android device.  

Can you help me to figure out what is wrong?  

Thanks.

**Comment by andyholmes** [comment]
The first thing I would try is force-stopping the Android app. Sometimes I think the sftp server gets stuck exiting, which it should do after each disconnect.

**Comment by dln949** [comment]
Thank you for helping.  

Sorry - I do not know what is meant by the term "force-stopping".  

I did try this:  Powered off the android tablet; shutdown the computer; restart the tablet; restart the computer.  Same problem still exists.

**Comment by andyholmes** [comment]
Oh, usually if you long-press the Android icon for KDE Connect you can go to "App Info" and there's a button to kill the app.

Was your Android always Oreo or did you just update?

**Comment by dln949** [comment]
I tried the "force-stopping" - that did not help.  

This tablet was at Oreo for at least a month, but the problem I am describing only started yesterday.  

I decided to try this:  I unpaired the tablet; I deleted the tablet from gsconnect on the desktop computer; I uninstalled kde connect from the tablet; I reinstalled kde connect on the tablet; finally, I paired the tablet and the desktop computer.  Now it seems to be working fine, the mounting problem appears to be fixed.  

I don't know what you think of all that, if that still indicates a problem that you wish (with my meager help, if you'd like) to further diagnose, or if you think, Well, if it works, great, let's say "case closed".  So, I will leave it up to you if you wish to close this.
